### PR TITLE
Swagger changes for 26 RFI connectors, each with a linkedservice, dataset and source

### PR DESCRIPTION
**Pull request by Eva-Xiao**
1. each connector has a linkedservice with specific properties apply to them
2. each connector has a dataset without any property
3. each connector has a copysource with a query property. 
4. app model and description is reviewed with PM

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by Eva-Xiao** [comment]
@hvermis

**Comment by olydis** [comment]
@ravbhatnagar lots of new stuff

**Comment by ravbhatnagar** [comment]
As discussed in skype call and over email, this is signed off from ARM side.

**Review by olydis — APPROVED** [review]
@Eva-Xiao Looks good to me, see comment about descriptions. I don't treat that as blocking since:
- you seem to have strict deadlines
- the documentation "pattern" seems to be nothing new in this Swagger
- adjusting documentation later is not a breaking change

Still, I'd appreciate it if you could clarify - maybe it's a low hanging fruit to improve customer experience! 🙂

File: specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/LinkedService.json [hunk 1]
```diff
@@ -1909,6 +1909,1655 @@
...
+      "required": [ 
+        "typeProperties" 
+      ] 
+    }, 
+    "AmazonMWSLinkedServiceTypeProperties": { 
+      "description": "Amazon Marketplace Web Service linked service properties.", 
+      "properties": {
+        "endpoint": {
+          "description": "The endpoint of the Amazon MWS server, (i.e. mws.amazonservices.com)",
+          "type": "object"
```
[the remark this answers is not in the document]
Comment (Eva-Xiao) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/LinkedService.json]: Thanks for pointing this out. Indeed I found it myself when I first start making this change, all the properties are object type except for the situation where it has only limited number of values, or must be a specific type for security concern. I think this is for future extensibility that a property can be extended to an object contains more properties where service can handled it without client breaking change.

**Pull request merged**